mifosng-android/src/main/res/values/strings.xml

```diff
@@ -394,6 +394,7 @@
 
     <!--Material Dialog Titles-->
     <string name="sync_previous_transaction">Sync Previous Transaction</string>
+    <string name="no_survey">No Survey Available for this client</string>
```

Comment:
please choose more meaningful names of your strings.

no_survey ->  no survey available for this client (Is quite ambiguous for some other dev)

no_survey_available_for_client (this would be more meaningful)